warehouse/deltalake/deltalake.go

```diff
@@ -668,10 +678,35 @@ func (dl *HandleT) connectToWarehouse() (*databricks.DBHandleT, error) {
 	return dl.connect(credT)
 }
 
+// getExternalLocation returns external location where we need to create the tables
+func (dl *HandleT) getExternalLocation() (externalLocation string) {
+	enableExternalLocation := warehouseutils.GetConfigValueBoolString(EnableExternalLocation, dl.Warehouse)
+	if enableExternalLocation == "true" {
+		externalLocation := warehouseutils.GetConfigValue(ExternalLocation, dl.Warehouse)
+		return externalLocation
+	}
+	return
+}
+
+// getTableLocationSql returns external external table location
+func (dl *HandleT) getTableLocationSql(tableName string) (tableLocation string) {
+	externalLocation := dl.getExternalLocation()
+	if externalLocation == "" {
+		return
+	}
+	return fmt.Sprintf("LOCATION '%s/%s'", externalLocation, tableName)
+}
+
 // CreateTable creates tables with table name and columns
 func (dl *HandleT) CreateTable(tableName string, columns map[string]string) (err error) {
 	name := fmt.Sprintf(`%s.%s`, dl.Namespace, tableName)
-	sqlStatement := fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s ( %v ) USING DELTA;`, name, columnsWithDataTypes(columns, ""))
+	tableLocation := dl.getTableLocationSql(name)
+	var partitionedSql string
```

Comment:
received_at.

Reply:
Even if we are switching on/off the enableExternal location, we are not clearing anything.